skalvi/views.py

```diff
@@ -100,6 +100,18 @@ def signOfEvent(request):
         # Will never happen
         return HttpResponse(status=206)  # not logged in
 
+def getAttendingActivities(request):
+    profile_name = request.path.split("/")[3]
+    profile = UserProfile.objects.get(user=request.user, profile_name=profile_name)
+    activities = ParticipateIn.objects.filter(userId=request.user, user_profile_id=profile)
+    activitie_objects = []
+    for activity in activities:
+        act = Activity.objects.get(pk=activity.activityId.pk)
+        activitie_objects.append(act)
+    json_serializer = serializers.get_serializer("json")()
+    attendingActivities = json_serializer.serialize(activitie_objects, ensure_ascii=False)
+    return HttpResponse(attendingActivities, content_type='application/json')
+
 
 def getActivities(request):
```

Comment:
Does this return a view, or fills out a view?
if the latter, extract to skalvi/ApiFunctions.py and then in urls.py where u use it, change to ApiFunctions.getActivities ?